staging/cert-manager-setup/templates/crds.yaml

```diff
@@ -0,0 +1,7514 @@
+
```

Comment:
i'm curious, why aren't use using the official chart and putting this as a subchart?

Reply:
Answer below :)